docs/source/terms_and_model.rst

```diff
@@ -422,7 +419,7 @@ $$$$$$$$$$$$$$$$$$
 
 **Biological definition**
 
-Imprecise chromosomal locations based on named landmarks.
+Chromosomal locations based on named landmarks.
```

Comment:
Made this change to drop "Imprecise", based off of [comments](https://github.com/ga4gh/vr-spec/pull/203#discussion_r447585697) from myself, @mbaudis, and a +1 from @rrfreimuth.

Minor, but as long as I'm making a revision PR...

Reply:
Yep. Sorry I missed this.